src/overlays/actors/ovl_Obj_Lift/z_obj_lift.c

```diff
@@ -30,29 +32,181 @@ const ActorInit Obj_Lift_InitVars = {
     (ActorFunc)ObjLift_Update,
     (ActorFunc)ObjLift_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B960D0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B960D8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B96160.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/ObjLift_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/ObjLift_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B9651C.s")
+s16 D_80B969C0[] = { 0x0000, 0x000A, 0x0014, 0x001E, 0x0028, 0x0032, 0x003C, 0x0000 };
 
```

Comment:
these are timer values so they make more sense in decimal. the last 0 entry is also padding so it can be omitted.
the name suggestion you dont have to use if you dont want, things will get documented later. but i think it fits
```suggestion
s16 sFallTimerDurations[] = { 0, 10, 20, 30, 40, 50, 60 };

```